### DISCUSSION TITLE
Dynatrace Secret Scanning rules

### DISCUSSION THREAD

**Discussion by Halarious** [community/community#43852] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

So a question. Dynatrace is part of the Github secret scanning program some time now (mentioned here https://github.blog/changelog/2020-07-22-our-secret-scanning-program-adds-five-new-partners/), but recently I noticed that Github makes mention of 'Dynatrace Internal Tokens' (see here https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/secret-scanning-patterns).

These are not documented as far as I can tell, and since Github distinguishes them from regular access tokens, I presume they follow a different format than that documented on the Dynatrace site?
Is there any indication on where these are used and how they can be exposed?

**Comment by marzvrover** [comment]
Hi @Halarious, they are indeed different types of tokens. I recommend reaching out to Dynatrace to learn more about the differences.

**Reply by Halarious** [reply]
Hey, thanks for the response. Dynatrace support sent me to Github for clarification 😅 I'm just interested in the differences in the format since they are distinguished and presumably able to be exposed somehow.

I'll also try Dynatrace again to see if I get better info this time.
